ext/pdo/pdo_stmt.c

```diff
@@ -2271,6 +2271,8 @@ static zval *row_prop_read(zend_object *object, zend_string *name, int type, voi
 			if (lval >= 0 && lval < stmt->column_count) {
 				fetch_value(stmt, rv, lval, NULL);
 			}
+		} else if (zend_string_equals_literal(name, "queryString")) {
+			return zend_std_read_property(&stmt->std, name, type, cache_slot, rv);
```

Comment:
Just from a style perspective, I'd move this before the is_numeric_string check.

I've also applied https://github.com/php/php-src/commit/7e3840df90db038de89f0a5041fac2f08497923d to exclude the null return value if stmt is unset.